app/Models/CampaignDashboardWidget.php

```diff
@@ -457,4 +457,25 @@ public function visible(): bool
         // Linked but no entity or no child? Permission issue or deleted entity
         return !empty($this->entity);
     }
+
+    public function filteredLink(string $entityString = null): string
+    {
+        $link = route('characters.index', [$this->campaign_id]);
```

Comment:
Feel like it will crash if the entity type is set to all :-)
![image](https://github.com/owlchester/kanka/assets/3263440/113a4b02-c06a-4b03-8846-8c260bb3fb1f)